$OpenBSD: patch-kio_kio_ksambashare_cpp,v 1.2 2015/12/23 10:31:43 zhuk Exp $
Allow to detect Samba presence. Should fine better way before
sending the patch upstream.
--- kio/kio/ksambashare.cpp.orig	Sun Jan 26 06:51:21 2014
+++ kio/kio/ksambashare.cpp	Sun Jan 26 06:52:58 2014
@@ -70,6 +70,7 @@ KSambaSharePrivate::~KSambaSharePrivate()
 bool KSambaSharePrivate::isSambaInstalled()
 {
     if (QFile::exists("/usr/sbin/smbd")
+        || QFile::exists("${LOCALBASE}/sbin/smbd")
         || QFile::exists("/usr/local/sbin/smbd")) {
         return true;
     }
